aat/src/aat/resources/features/F-107 CaseLink/S-107.1_Get_CaseType.td.json

```diff
@@ -0,0 +1,3171 @@
+{
+  "_guid_": "S-107.1_Get_CaseType",
+  "_extends_": "F-107_Get_CaseType",
+
+  "specs": [
+    "fetch Master Case Type to verify CaseLink and LinkReason CaseField base types"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "id": "FT_MasterCaseType"
+    }
+  },
+
+  "expectedResponse": {
+    "body": {
+      "id": "FT_MasterCaseType",
+      "case_fields": [
+        {
+          "metadata": false,
+          "id": "TextField",
+          "case_type_id": "FT_MasterCaseType",
+          "label": "Text Field",
+          "hint_text": null,
+          "field_type": {
+            "id": "Text",
+            "type": "Text",
+            "min": null,
+            "max": null,
+            "regular_expression": null,
+            "fixed_list_items": [ ],
+            "complex_fields": [ ],
+            "collection_field_type": null
+          },
+          "hidden": false,
+          "security_classification": "PUBLIC",
+          "live_from": "2017-01-01",
+          "live_until": null,
+          "acls": [
+            {
+              "role": "caseworker-befta_master",
+              "create": true,
+              "read": true,
+              "update": true,
+              "delete": true
+            }
+          ],
+          "complexACLs": [ ],
+          "order": null,
+          "show_condition": null,
+          "display_context_parameter": null,
+          "retain_hidden_value": null
+        },
+        {
+          "metadata": false,
+          "id": "TextAreaField",
+          "case_type_id": "FT_MasterCaseType",
+          "label": "Text Area",
+          "hint_text": null,
+          "field_type": {
```

Comment:
Feel free to set some of the more complex properties to `"[[ANYTHING_PRESENT]]"` on those extra fields we are not reviewing in this test.  e.g. `field_type`, `acls` on fields that are not `CaseLink` or `LinkReason`.

Reply:
Done in [bcc54b2](https://github.com/hmcts/ccd-definition-store-api/pull/1180/commits/bcc54b212c1c01e12253ae4449ee470f6bd6f611)